### PR TITLE
Robotic legs let you use pockets without a jumpsuit, robotic chest lets you wear an ID/belt without jumpsuit

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Robotic legs now let you use pockets without a jumpsuit, and a robot chest now lets you use the belt slot and ID slot without a jumpsuit.
/:cl:

Why: So people can show off their robotic limbs, as they are about to be resprited

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -665,7 +665,10 @@
 		if(slot_belt)
 			if(H.belt)
 				return 0
-			if(!H.w_uniform && !nojumpsuit)
+
+			var/obj/item/bodypart/O = H.get_bodypart("chest")
+
+			if(!H.w_uniform && !nojumpsuit && (!O || !O.status == BODYPART_ROBOTIC))
 				if(!disable_warning)
 					to_chat(H, "<span class='warning'>You need a jumpsuit before you can attach this [I.name]!</span>")
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/species.dm]: >!O

you need a CHEST before you can attach this! :P

**Comment by RandomMarine** [comment]
robocop_01
Augmented pockets holding guns when?

**Comment by RemieRichards** [comment]
@RandomMarine you joke but technically we could increase the supported "weight class" (```w_class```) of pockets if their legs are auged.

**Comment by KorPhaeron** [comment]
@RandomMarine 

When you put a stetchkin in your leg

**Comment by InsaneHyena** [comment]
Darth Vader hype

**Comment by Ausops** [comment]
is that a stun baton in your pants or are you just happy to s-*whack*

**Comment by RandomMarine** [comment]
Technically you could but the balance memers would come out in full force.

**Comment by Anonmare** [comment]
Augmented bodyparts are getting re-sprited?

**Comment by GunHog** [comment]
@InsaneHyena is commisioning a set for $25 USD: https://tgstation13.org/phpBB/viewtopic.php?f=11&t=11854&view=unread#p311317 (Why does he put the $ after the number? That slightly irritates me grrrrrrr)

**Comment by InsaneHyena** [comment]
That's the way we write it in Russia.

**Comment by Qbopper** [comment]
re: pocket guns

on the one hand, s t o r a g e c r e e p

on the other that's a really fucking cool idea

**Comment by XDTM** [comment]
make leg storage an implant

**Comment by loqoman** [comment]
I feel like the another 'why' reasons should be added along the lines of...

'It makes sense'

**Comment by optimumtact** [comment]
cheridan in here to rip you a new one about storage creep shortly

**Comment by KorPhaeron** [comment]
@optimumtact You can't hold any extra things with this PR, though I agree stuffing full sized items into your legs would certainly be storage creep

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -665,7 +665,10 @@
 		if(slot_belt)
 			if(H.belt)
 				return 0
-			if(!H.w_uniform && !nojumpsuit)
+
+			var/obj/item/bodypart/O = H.get_bodypart("chest")
+
+			if(!H.w_uniform && !nojumpsuit && (!O || !O.status == BODYPART_ROBOTIC))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species.dm]: Does this line really do what you think it does?

Hint: It doesn't

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -705,7 +708,9 @@
 		if(slot_wear_id)
 			if(H.wear_id)
 				return 0
-			if(!H.w_uniform && !nojumpsuit)
+
+			var/obj/item/bodypart/O = H.get_bodypart("chest")
+			if(!H.w_uniform && !nojumpsuit && (!O || !O.status == BODYPART_ROBOTIC))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species.dm]: Again

File: code/modules/mob/living/carbon/human/species.dm [hunk 4]
```diff
@@ -717,7 +722,10 @@
 				return 0
 			if(H.l_store)
 				return 0
-			if(!H.w_uniform && !nojumpsuit)
+
+			var/obj/item/bodypart/O = H.get_bodypart("l_leg")
+
+			if(!H.w_uniform && !nojumpsuit && (!O || !O.status == BODYPART_ROBOTIC))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species.dm]: Ditto

File: code/modules/mob/living/carbon/human/species.dm [hunk 5]
```diff
@@ -730,7 +738,10 @@
 				return 0
 			if(H.r_store)
 				return 0
-			if(!H.w_uniform && !nojumpsuit)
+
+			var/obj/item/bodypart/O = H.get_bodypart("r_leg")
+
+			if(!H.w_uniform && !nojumpsuit && (!O || !O.status == BODYPART_ROBOTIC))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species.dm]: More of the same

File: code/modules/mob/living/carbon/human/species.dm (hunk 2 above)
Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/species.dm]: I put the ! in the wrong place whoops

**Comment by MoreRobustThanYou** [comment]
Maybe I can start maining RD again

**Comment by KorPhaeron** [comment]
@Cyberboss

**Pull request merged**